docs/src/modules/ROOT/attachments/akka-code-init.json

```diff
@@ -14,6 +14,13 @@
       "url": "https://github.com/akka-samples/empty",
       "level": "Beginner"
     },
+    {
+      "title": "Empty project (Kotlin)",
+      "description": "This project contains the build and project structure to create an Akka service in Kotlin.",
+      "language": "kotlin",
+      "url": "https://github.com/akka-samples/empty-kotlin",
+      "level": "Beginner"
+    },
```

Comment:
This is adding it just after the java empty one. 

I wonder why the first is not the empty. I guess was discuss already, but feel strange, IMO.